lib/internal/bundle.js

```diff
@@ -80,7 +80,7 @@ async function buildInputOptions(props, options, htmlConfig) {
   const tplSv = tplFile.replace(".html", ".svelte");
 
   const tplJs = `
-import App from '${tplSv}';
+import App from '${JSON.stringify(tplSv)}';
```

Comment:
I think we need to remove this single quote, since `JSON.stringify` will wrap the path using double quote.

```suggestion
import App from ${JSON.stringify(tplSv)};
```

Reply:
Good catch